src/gridstack.js

```diff
@@ -1155,12 +1162,12 @@
 
   GridStack.prototype._updateContainerHeight = function() {
     if (this.engine._batchMode) { return; }
-    var row = this.engine.getRow();
+    var row = this.opts.minRow > this.engine.getRow() ? this.opts.minRow : this.engine.getRow();
```

Comment:
don't call methods twice! inefficient. either put the minRow in getRow() (see where it's used or here)